static/less/group-detail.less

```diff
@@ -702,7 +701,6 @@ div.traceback > ul {
       > li.active {
         position: relative;
         z-index: 0;
-        color: #fff;
```

Comment:
We really need to delete this whole file, but tryna keep this project short and sweet.

Reply:
2 lines down, 1177 more to go